Modules/Filtering/LabelMap/test/itkShapeLabelMapFilterGTest.cxx

```diff
@@ -429,6 +444,46 @@ TEST_F(ShapeLabelMapFixture, 2D_T1_1_FlipDirection)
 }
 
 
+TEST_F(ShapeLabelMapFixture, 2D_T1_2_Direction)
+{
+  using namespace itk::GTest::TypedefsAndConstructors::Dimension2;
+
+  using Utils = FixtureUtilities<2>;
+
+  Utils::ImageType::Pointer image(Utils::CreateImage());
+
+  image->SetPixel(MakeIndex(5, 7), 1);
+  image->SetPixel(MakeIndex(5, 8), 1);
+
+  DirectionType direction;
+
+  const double d[4] = { 0, 1.0, 1.0, 0 };
+
+  direction = DirectionType::InternalMatrixType(d);
+
+  image->SetDirection(direction);
+
```

Comment:
Too much white-space for my taste. An empty line between each statement does not really make them stand out more.

Reply:
This is compliant with our clang formatting configuration and ITK coding standards.

Reply:
It is compliant, but so would @dzenanz suggestion.  You could also put 0 lines, or 2 lines between each statement, and that would also be compliant.  3 lines would not be compliant.